### DISCUSSION TITLE
Exported video is the wrong resolution and colours are faded, what do I do now?

### DISCUSSION THREAD

**Discussion by bertvandepoel** [senshu/Sozi#624] [Q&A]
Hi there! I'm new to Sozi but have been using Inkscape for years, and was recommended Sozi since it works together so well with Inkscape. I'm making a basic animation (some text or elements that grow, we're not sure yet exactly where we're going).

The project is made on 9:16 (since the final resolution will be 1080x1920, a standard screen on its side basically). Weirdly enough, when I export to video and even explicitly enter this resolution, the resulting file is a totally different resolution, namely 1059x1119. I have no idea where it's getting this resolution from, it seems completely random. When I export to PNG, the colours look fine, just a weird resolution, but if I export to a video, the colours also look really faded. 

I'm getting quite frustrated trying to figure out why this keeps happening. I hope someone here knows what to do.

EDIT: It seems that this is partially caused by the desired end resolution being larger than my screen. However, even on a 4K screen, it renders a completely different resolution, namely 1920x2381.

**Comment by aumouvantsillage** [comment]
> Weirdly enough, when I export to video and even explicitly enter this resolution, the resulting file is a totally different resolution, namely 1059x1119.

I have noticed a similar issue. In my case, it was related to the display resolution of the computer running Sozi.

During the export process, Sozi plays the presentation in a separate window and makes screenshots of this window.
When the window is too big to fit on the screen, I assume that it is automatically resized.

At the moment, I don't have a real solution to this problem. The only workarounds are:

* Use a bigger display.
* Export to a lower resolution.

**Reply by aumouvantsillage** [reply]
I have just seen your EDIT. I will try to find a way to debug the problem.

**Comment by aumouvantsillage** [comment]
> However, even on a 4K screen, it renders a completely different resolution, namely 1920x2381.

Can you confirm that you are setting the same width and height as in the previous attempts (width = 1920, height = 1080)?
I don't see a reason why the exported video should be bigger than the desired size.

**Reply by bertvandepoel** [reply]
I asked my friend with the 4K screen several times to confirm, since I thought it was weird too. So yeah, it does seem to be the case :/
